src/emulator.js

```diff
@@ -87,6 +88,22 @@ export class Emulator {
     this.logProcessor = (item) => item;
 
     return new Promise((resolve, reject) => {
+      let internalId;
+      const checkLiveness = function() {
+        const url = "http://localhost:" + port + "/live"
```

Comment:
We can string template literals here: 
```suggestion
        const url = `http://localhost:${port}/live`
```